### ISSUE TITLE
[Accessibility] | Chart Component in Storybook throwing img needs alt text violoation

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#11558]
## Issue Description
// What is the nature of the issue? How does it impact users?
More info here : https://dequeuniversity.com/rules/axe/4.10/role-img-alt?application=axeAPI

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question
Both Line and Doughnut components have the same issue
https://6555d3db80418bb1681b8b17-ssvavnslez.chromatic.com/?path=/story/components-chart--line
https://6555d3db80418bb1681b8b17-ssvavnslez.chromatic.com/?path=/story/components-chart--doughnut

## What application are you using?
OJS, OMP, OPS 3.5.0

## Stack used 
- Device: chrome browser
- OS: Windows 10
- Browser: Chrome
- Version: 130

## Testing method
- Method: (e.g. Manual; Keyboard)
- Automated test: Storybook with Ally Plugin

## Action
// Describe action that can be taken to remediate the issue. 
Add appropriate alt text for image

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![Image](https://github.com/user-attachments/assets/3fe217b6-12e5-43a3-ae41-cbf53eb565ed)

## Pull Requests

- ui-library 3.5: https://github.com/pkp/ui-library/pull/670
- ui-library main: https://github.com/pkp/ui-library/pull/669
- pkp-lib 3.5: https://github.com/pkp/pkp-lib/pull/11661
- pkp-lib main: https://github.com/pkp/pkp-lib/pull/11662
- OJS 3.5: https://github.com/pkp/ojs/pull/5004 
- OJS main: https://github.com/pkp/ojs/pull/5005

**Comment by kaitlinnewson** [comment]
I discussed this issue with @blesildaramirez, and the likely fix will be to change Storybook so that the markup is more similar to what is in the applications, particular the parent div for the chart which contains `aria-hidden=true`. Once that is changed in Storybook, then this issue should no longer be flagged.

**Comment by Tribunal33** [comment]
~I think Jarda fixed this.~
I think Jarda had a fix for this.  Same idea.

**Comment by jardakotesovec** [comment]
I just confirmed what Kaitlin is saying here. Adding that aria-hidden in storybook is good way to address that.

**Comment by kaitlinnewson** [comment]
Hi @blesildaramirez, can you review the small PRs above? Thanks!

**Comment by Tribunal33** [comment]
This passes in storybook now.  No longer seeing accessibilty violations.  Please close once everything else is merged @kaitlinnewson .

<img width="566" height="121" alt="Image" src="https://github.com/user-attachments/assets/211ecdc4-e536-444e-aea5-9877ecb5f8ad" />

**Comment by kaitlinnewson** [comment]
Hi @taslangraham, could you do the second review of the small PRs above? It's already merged for the UI library, so just some cleanup to do in the applications. Thanks!

**Comment by taslangraham** [comment]
Looks good, @kaitlinnewson! I’ve approved all the PRs.

There are a few failing tests on the App PRs that need to be looked at, but I don’t think they’re related to your changes.
